specification/communication/Communication.Messages/routes.tsp

```diff
@@ -0,0 +1,37 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "./models.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace AzureCommunicationMessagesService;
+
+alias ServiceTraits = SupportsRepeatableRequests &
+  NoConditionalRequests &
+  SupportsClientRequestId;
+
+alias Operations = Azure.Core.ResourceOperations<ServiceTraits>;
+
+interface TemplateOperations {
+  @tag("Template")
+  @doc("List all templates for given ACS channel")
+  getTemplates is Operations.ResourceList<MessageTemplateItem>;
+}
+
+interface NotificationMessagesOperations {
+  @tag("Notifications")
+  @doc("Sends a notification message from Business to User.")
+  send is Operations.ResourceCollectionAction<Notifications, NotificationContent, AcceptedResponse & SendMessageResult>
+}
+
+interface StreamOperations {
+  @tag("Stream")
+  @doc("Download the Media payload from a User to Business message.")
```

Comment:
Sometimes we use "media", and sometimes "stream" to describe the same thing. Can we look up from prior reviews why we named this /streams, and double-check that we're consistent in our terminology?

Reply:
The operation namespace is Stream, and the operation is called downloadMedia.

Reply:
Discussed in the review meeting, will keep it this way for now, until we can come up with a better name.